src/audio-worklet/free-queue/src/interface/C/free_queue.h

```diff
@@ -0,0 +1,206 @@
+#ifndef FREE_QUEUE_C
```

Comment:
Please revise this file with Google C++ style guide:
https://google.github.io/styleguide/cppguide.html